website/blog/2023-08-01-announcing-materialized-views.md

```diff
@@ -126,7 +131,7 @@ config(
 }}
 ```
 
-For Redshift:
+For Redshift/BigQuery:
```

Comment:
The snippet would be different for BiqQuery; the input parameters follow BQ parameters. It would look something like this:
```sql
{{ config(
    materialized = "materialized_view",
    on_configuration_change = "apply",
    enable_refresh = True,
    refresh_interval_minutes = 30,  # optional for `enable_refresh`, defaults to 30
    max_staleness = 60,  # optional for `enable_refresh`, defaults to `None`
) }}
```

Reply:
Oh I didn't realize reference_interval_minutes was required

Reply:
`refresh_interval_minutes` is not required, nor is `max_staleness`. Even `enable_refresh` is not required, it defaults to `True`.